### PR TITLE
DM-11332: FITS compression

### PR DESCRIPTION
**Pull request by PaulPrice**

### CODE REVIEW

File: patches/cfitsio-3360-make-install.patch [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+-install:	libcfitsio.a $(INSTALL_DIRS)
+-		/bin/cp -a libcfitsio* ${CFITSIO_LIB}
+-		/bin/cp fitsio.h fitsio2.h longnam.h drvrsmem.h ${CFITSIO_INCLUDE}
+-		/bin/cp cfitsio.pc ${CFITSIO_LIB}/pkgconfig
++install: libcfitsio.a ${INSTALL_DIRS}
++	@for lib in libcfitsio.a libcfitsio${SHLIB_SUFFIX} \
++		${CFITSIO_SHLIB} ${CFITSIO_SHLIB_SONAME}; do \
++	    if [ -f $$lib ]; then \
++	    echo "/bin/cp -a $$lib ${CFITSIO_LIB}"; \
++	    /bin/cp -a $$lib ${CFITSIO_LIB}; \
```
Comment (parejkoj) [inline patches/cfitsio-3360-make-install.patch:37]: I don't like that this has `/bin/cp`, but that probably can't be helped...

File: patches/cfitsio-3360-make-install.patch [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
++	    fi; \
++	done
++	/bin/cp fitsio.h fitsio2.h longnam.h drvrsmem.h ${CFITSIO_INCLUDE}
++	/bin/cp cfitsio.pc ${CFITSIO_LIB}/pkgconfig
```
Comment (parejkoj) [inline patches/cfitsio-3360-make-install.patch:41]: Worth `echo`ing these `cp`s as well?

**Review by parejkoj — COMMENTED** [review]
Two short comments. Only other question is if these patches have been submitted upstream?

**Comment by PaulPrice** [comment]
The two "fix" patches have been submitted upstream. The "fix false compression error" may be fixed differently upstream when the release comes out, I'm not sure.
The changes to the Makefile are a backport from cfitsio 3.41 (so I'm not responsible for those `/bin/cp`s, and I'd prefer not to change anything unless there's a demonstrated problem). I'll make that clear in the commit message.

**Pull request merged**